apps/gdalinfo_lib.cpp

```diff
@@ -382,8 +394,27 @@ char *GDALInfo( GDALDatasetH hDataset, const GDALInfoOptions *psOptions )
         if( bJson )
         {
             json_object *poWkt = json_object_new_string(pszPrettyWkt);
+            if (strcmp(psOptions->pszWKTFormat, "WKT2") == 0)
+            {
+                json_object *poStacWkt = nullptr;
+                json_object_deep_copy(poWkt, &poStacWkt, nullptr);
+                json_object_object_add(poStacProperties, "proj:wkt2", poStacWkt);
+            }
             json_object_object_add(poCoordinateSystem, "wkt", poWkt);
 
+            const char *epsg = OSRGetAuthorityCode(hSRS, nullptr);
+            if (epsg)
+            {
+                json_object *poEPSG = json_object_new_int64(atoi(epsg));
```

Comment:
there can be codes in other namespaces than EPSG:, so:
```suggestion
            const char *pszAuthCode = OSRGetAuthorityCode(hSRS, nullptr);
            const char *pszAuthName = OSRGetAuthorityName(hSRS, nullptr);
            if (pszAuthCode && pszAuthName && EQUAL(pszAuthName, "EPSG"))
            {
                json_object *poEPSG = json_object_new_int64(atoi(pszAuthCode));
```